.expeditor/scripts/validate_adhoc_build.ps1

```diff
@@ -1,21 +1,36 @@
 $ErrorActionPreference = 'Stop'
 
+
```

Comment:
wait, I thought this and validate_adhoc_build.sh were replaced by hooks/validate-adhoc.rb?

Reply:
@jaymzh The old validate_adhoc_build.sh bash script was only generating the configuration file that gets uploaded to buildkite. That generated pipeline configuration points to these scripts to run the actual builds and tests. We changed that generation script to ruby which is what validate-adhoc.rb contains.